src/App.tsx

```diff
@@ -68,12 +72,19 @@ const useStyles = makeStyles((theme: Theme) =>
       }),
       marginLeft: 0,
     },
+    resetButton: {
+      margin: 20,
+    },
   })
 );
 
 export default function App() {
   const classes = useStyles();
   const [open, setOpen] = React.useState(true);
+  const filesUploaded = useSelector(
+    (state: UploadState) => state.filesUploaded
```

Comment:
Według mnie, lepiej nazywać flagi boolenowskie dodając przefix `is`, ponieważ to dość jawnie wskazuje na ich funkcję. 
Natomiast kiedy masz samo `filesUploaded` dla mnie to wygląda jak nazwa zmiennej przechowującej wczytane pliki.